compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -48,7 +48,8 @@ class InvalidStateError(Exception):
     from pika.frame import Method
     from pika.spec import Basic, BasicProperties
 
-
+_API_BURST_LIMIT = 5
```

Comment:
I'd get value out of explaining in code how these affect usage from user's point of view.  ie. what's in the changelog repeated here, a short summary?  Instead of calculating the 5.5s window after initial burst etc.

Reply:
I think this is addressed by the push I just made?  Lemme know.  Key details:

- now user-configurable
- accordingly, the two parameters are documented
- only API calls that are made without filling the batch are counted toward the burst limit

Reply:
Great.  As we aren't settled yet on long term rate limiting strategies, this isn't worth also adding to our readthedocs yet, but eventually we should have a section on usage recommendations.